### ISSUE TITLE
Cords have completed and checked information held on each syndicate member

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/FreeCENMigration#1066]
Story created as in progress, and present as a story on Roadmap (was #87 which was to implement capability to record the information

**Comment by geoffj-FUG** [comment]
Pat
Bev’s volunteers are not even registered and she has declined to register them or use CSVProc. She has decided that her volunteers are different to everybody else’s as they wont cope online.
In addition she has recently added a County to her workload so she now has Cambridge / Norfolk / Suffolk / Surrey / Essex. That is 5 Counties that are presumably not registered and which she has refused to link the PARMS for.
On that basis this story cannot be completed and the Development Team will have wasted the past 9 months!
This requires management action. I can continue to action these Counties right up to implementation, but I do not have the authority to make a direction.
Geoff

**Comment by FreecenBren** [comment]
Not strictly true.
She does have members registered. I have just looked at a couple of her
counties.
What she is saying is that many or her helpers, want to carry on as they do
now.
She was assured that they could by both Pat and Kirk.

It is very easy to read what COORDS have been sent to do with CSVProc, but
without the required training  it is not easy.
You can only follow instructions to a certain point then something happens
and how to get back to where you had got to is not easy.

I did hear mention that there was to be a video for COORDS to watch while
reading the instructions and what to do in FC2.
Was that my imagination or did I miss the video.

I do sympathise with Bev as many of my helpers feel the same.
I also know that quite a few COORDS think likewise.

It is ok to get all these things written  but there is nothing that can
substitute training.
Reading a 114+  Pages of instructions  is a nightmare. You get lost along
the way.
Whereas a Video to watch while reading it, would be really beneficial.
You can then turn the video back and read it again to ensure you have it
right.

Was I just dreaming about a Video or have I missed it along the road.

Cheers Brenda

**Comment by Captainkirkdawson** [comment]
Brenda; 
New volunteers are automatically registered. However little use has been made of that fact for the very good reason that there was no benefit to so doing. 
What I said was that as a volunteer organization we cannot force people to change and some find change difficult. We can encourage, cogole and support those that are willing to try.
It is important to remember, as you know better than most, that FreeCEN is extremely old technology in all aspects. We have been more than extremely lucky that our offline software continues to run; it will not for much longer I am afraid.  It has been kept alive by the work of a fewer and fewer volunteers. When you and Lemon are unwilling or unable to do what you do all updating will cease and all the thousands of hours of work will go to waste.
We went through a similar experience with Freereg 4 years ago; there was an initial reluctance to the change but it did not take long for people to see the benefits and the number of volunteers has increased dramatically. So that the database has doubled in size in four years.
Yes the handbook is large and appears daunting but vast quantities are of a reference nature. I know @PatReynolds talked about the usefulness of training videos and what components there could be there are no active stories that would lead to their generation unless I have missed something.
The bottom line IMO is that we need you to join in the implementation and encourage the adoption of these new practices. This will ensure that your years of work and considerable commitment will result in a living legacy that continues to grow indefinitely.

**Comment by PatReynolds** [comment]
Hello all, do you think it would be helpful to coords (and others) if Anna our video/training volunteer were to make a short video explaining the buttons and what they do?  e.g. "this is where you go if you want to send a message to your coordinator, or to any other member of the FreeCEN project'.? @geoffj-FUG @FreecenBren

**Comment by FreecenBren** [comment]
I am sure that would be very useful. 
 
Any help that COORDS can get to help with the future move over to FC2  would be most welcome.

**Comment by DeniseColbert** [comment]
Story made: https://github.com/freeukgen/freecenmigration/issues/1078

Anna and I will work on this

**Comment by geoffj-FUG** [comment]
Denise
Be aware that at the moment volunteers send emails to their Coordinator. We should not change that at the moment - too much, too soon. We need to get people workingwith the essential basics first.
Geoff

**Comment by DeniseColbert** [comment]
We are planning a future video on Communications but it will be a general video, as all three project websites have the same system. This can be shared with FreeCEN transcribers at the appropriate time.

**Comment by PatReynolds** [comment]
Coords are doing this as part of starting to use FreeCEN2.

Can this be closed @geoffj-FUG
